build/common.sh

```diff
@@ -23,27 +23,27 @@ USER_ID=$(id -u)
 GROUP_ID=$(id -g)
 
 DOCKER_OPTS=${DOCKER_OPTS:-""}
-DOCKER=(docker ${DOCKER_OPTS})
+DOCKER=(docker "${DOCKER_OPTS}")
```

Comment:
Gut feeling is that you _would_ want word splitting here into array items?

Reply:
Good catch, @stevekuznetsov.

/lgtm cancel

Reply:
Indeed, this wasn't a subshell invocation... Thanks Steve! Fixing this.